organizations/v0/tests/test_views.py

```diff
@@ -0,0 +1,67 @@
+"""
+Organizations Views Test Cases.
+"""
+from django.test import TestCase
+from django.core.urlresolvers import reverse
+
+from organizations.tests.factories import UserFactory, OrganizationFactory
+
+
+DRF_DATE_FORMAT = u'%Y-%m-%dT%H:%M:%S.%f'
+
+
+# pylint: disable=no-member, no-self-use
+class TestOrganizationsView(TestCase):
+    """ Test Organizations View."""
+    def setUp(self):
+        super(TestOrganizationsView, self).setUp()
+
+        self.user_password = 'test'
+        self.user = UserFactory(password=self.user_password)
+        self.organization = OrganizationFactory.create()
+        self.organization_url = reverse('v0:organization-list')
+
+    def serialize_organization(self, organization):
+        """Return object-to-dict representation of organization."""
+        return {
+            "id": organization.id,
+            "name": organization.name,
+            "short_name": organization.short_name,
+            "description": organization.description,
+            "logo": None,
+            "active": organization.active,
+            "created": organization.created.strftime(DRF_DATE_FORMAT),
+            "modified": organization.modified.strftime(DRF_DATE_FORMAT)
+        }
+
+    def _get_organization_url(self, short_name):
```

Comment:
In the rare event that we decide to use something other than the `short_name`, pass in the entire `Organization` object.